docs/csharp/quick-starts/arrays-and-collections.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Quick Start - Collections - C# Guide
+description: Learn C# by exploring the List collection in this quick start.
+keywords: C#, Get Started, tutorial, collections, List
+author: billwagner
+ms.author: wiwagn
+ms.date: 10/13/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+# C# Quick start: Collections #
+
+This tutorial provides an introduction to the C# language and the basics of the <xref:System.Collections.Generic.List%601>
+class.
+
+## A simple list example.
+
+> [!NOTE]
+> If you are starting from the code you wrote in [dot.net](https://dot.net/), you will already have the code written in this section. Jump to [Modify list contents](#modify-list-contents).
+
+
+Create a directory named **list-quickstart** and run `dotnet new console`.
```

Comment:
No prereqs? Do we just assume they have .NET Core 2.0 SDK installed?

Also, should we make it explicit that they run the command inside that new directory in a terminal window?